docs/config/microshift-starter.ks

```diff
@@ -46,13 +47,14 @@ selinux-policy-devel
 # Allow the default user to run sudo commands without password
 echo -e 'redhat\tALL=(ALL)\tNOPASSWD: ALL' > /etc/sudoers.d/redhat
 
-# Update selinux-policy packages from CentOS 8 Stream
-CENTOS8BASE=http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages
-curl -LO -s $CENTOS8BASE/selinux-policy-3.14.3-96.el8.noarch.rpm
-curl -LO -s $CENTOS8BASE/selinux-policy-devel-3.14.3-96.el8.noarch.rpm
-curl -LO -s $CENTOS8BASE/selinux-policy-targeted-3.14.3-96.el8.noarch.rpm
-dnf localinstall -y selinux-policy*.rpm
-rm -f selinux-policy*.rpm
+tee /etc/yum.repos.d/rhocp-4.12-el8-beta-$(uname -i)-rpms.repo >/dev/null <<EOF
+[rhocp-4.12-el8-beta-$(uname -i)-rpms]
+name=Beta rhocp-4.12 RPMs for RHEL8
+baseurl=https://mirror.openshift.com/pub/openshift-v4/\$basearch/dependencies/rpms/4.12-el8-beta/
+enabled=1
+gpgcheck=0
+skip_if_unavailable=1
```

Comment:
I'd set skip_if_unavailable=0, as skipping it on install/update may lead to follow-up issues and reporting an error here would be clearer.